.gitignore

```diff
@@ -1,3 +1,4 @@
+auth.json
```

Comment:
this should not be in the project gitignore. It is not projct-specific (btw, composer will never write in this file by default as it writes auth in the global file).

If you really want to use a local credential file, use a gitignore rule specific to your computer: https://help.github.com/articles/ignoring-files/